### PR TITLE
Changed subtitle from applications to search.

### PR DESCRIPTION
**Pull request by JacOng17**
# What has changed and why?
Changed subtitle from applications to search.
Cust Search

### CODE REVIEW

**Comment by JacOng17** [comment]
@dlouise64 Probably would, will go take a look now :+1:

**Comment by dlouise64** [comment]
@JacOng17 Also looks like you might need to run the tests :) 👍

File: app/javascript/components/CustomerSearch.js [hunk 1]
```diff
@@ -92,32 +92,32 @@ class CustomerApplications extends React.Component {
           </thead>
           <tbody className='rebate-results-table-body'>
             {map(this.state.rebateForms, (rebateForm, key) => {
-              const { property, fields, id } = rebateForm
+              const { property, fields, id } = rebateForm;
               return (
-                <tr key={`${key}-${fields.full_name}`} className='rebate_form.completed'>
+                <tr key={`${key}-${fields.full_name}`} className="rebate_form.completed">
```
Comment (jvdanker) [inline app/javascript/components/CustomerSearch.js]: style issue; on line 55 you used the `{'class'}` syntax which is different to this line

File: app/javascript/components/CustomerSearch.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
+
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/javascript/components/CustomerSearch.js]:
```suggestion
```

**Pull request closed without merging**